[gamemodes]/[play]/play/play.lua

```diff
@@ -0,0 +1,18 @@
+function onResourceStartPlay()
```

Comment:
Would a local function work?
```suggestion
local function onResourceStartPlay()
```